docs/index.rst

```diff
@@ -785,6 +785,14 @@ The SecretStr and SecretBytes will be formatted as either `'**********'` or `''`
 
 (This script is complete, it should run "as is")
 
+Strict Types
+............
+
+You can use the ``StrictStr``, ``StrictInt``, ``StrictFloat``, and ``StrictBool`` types to prevent coercion from compatible types.
+These types will only pass validation when the validated value is of the respective type or is a subtype of that type.
+This behavior is also exposed via the ``strict`` field of the ``ConstrainedStr``, ``ConstrainedFloat`` and
+``ConstrainedInt`` classes and can be combined with a multitude of complex validation rules (please note that there is no ``ConstrainedBool`` class).
```

Comment:
I'm not sure what `(please note that there is no ConstrainedBool class)` adds? Also, this section is about strict, not contstrained.

Reply:
A curious read may ask what about `ConstrainedBool`, so I pointed out that there is no such thing because what's there to constrain in `bool` except of instance checking (well, you can require it to be only `True` or only `False`, but I'm not sure about a use case for that).

Anyhow, if you feel it's unnecessary, I have no sentiments to this line :)

Reply:
I think it's pretty clear as it is, we explicitly list  the `Constrained*` types it works with.

Hopefully anyone asking themselves "why not `ConstrainedBool`?" will be able to answer it for themselves in the same breath.